core/domain/platform_feature_services.py

```diff
@@ -48,6 +48,11 @@
 ALL_FEATURES_NAMES_SET = set(ALL_FEATURES_LIST)
 
 
+class FeatureFlagNotFoundException(Exception):
+    """Error class for feature flag not found error."""
```

Comment:
Alternative suggestion: ``Exception thrown when an unknown feature flag is requested.`` Feel free to resolve this comment once addressed.

Reply:
Done.